continuous_integration/recipes/distributed/conda_build_config.yaml

```diff
@@ -1,3 +1,6 @@
 python:
     - 3.8
     - 3.9
```

Comment:
Should we remove this now that Python version is negligible?

```suggestion
```

Reply:
Possibly if we add a `skip`